.github/workflows/release.yml

```diff
@@ -73,11 +74,12 @@ jobs:
       image: jsii/superchain
     steps:
       - name: Download build artifacts
-        uses: actions/download-artifact@v1
```

Comment:
Shouldn't these be in the projen file somehow?

Reply:
What? the version of the download-artifact action? No... They are part of the projen component.